maps/aurora/aurora-3_sublevel.dmm

```diff
@@ -21140,7 +21140,9 @@
 	},
 /obj/effect/floor_decal/industrial/outline/blue,
 /obj/machinery/vending/wallmed1{
-	pixel_y = 32
+	pixel_y = 32;
+	req_access = null;
```

Comment:
Should be safe removing this line. By default `req_access` is just `list()`.

Reply:
```suggestion
```